platforms/common/px4_log.cpp

```diff
@@ -183,11 +184,11 @@ __EXPORT void px4_log_modulename(int level, const char *module_name, const char
 		log_message_s log_message;
 
 		const uint8_t log_level_table[] = {
-			7, /* _PX4_LOG_LEVEL_DEBUG */
-			6, /* _PX4_LOG_LEVEL_INFO */
-			4, /* _PX4_LOG_LEVEL_WARN */
-			3, /* _PX4_LOG_LEVEL_ERROR */
-			0  /* _PX4_LOG_LEVEL_PANIC */
+			LOG_DEBUG,    /* _PX4_LOG_LEVEL_DEBUG */
```

Comment:
I'm not sure if we can depend on this numbering being stable across Linux, MacOS, cygwin, etc.

Perhaps the correct thing to do would be put constants in https://github.com/PX4/PX4-Autopilot/blob/1ad83a80025c6ec719eb5c2ab2025619c7eff497/msg/LogMessage.msg#L5 ?